src/dense/verner_addsteps.jl

```diff
@@ -632,28 +632,30 @@ end
     copyat_or_push!(k,5,f(uprev+dt*(a0501*k[1]       +a0503*k[3]+a0504*k[4]),p,t+c4*dt))
     copyat_or_push!(k,6,f(uprev+dt*(a0601*k[1]                +a0604*k[4]+a0605*k[5]),p,t+c5*dt))
     copyat_or_push!(k,7,f(uprev+dt*(a0701*k[1]                +a0704*k[4]+a0705*k[5]+a0706*k[6]),p,t+c6*dt))
-    copyat_or_push!(k,8,f(uprev+dt*(a0801*k[1]                                  +a0806*k[6]+a0807*k[7]),p,t+c7*dt))
-    copyat_or_push!(k,9,f(uprev+dt*(a0901*k[1]                                  +a0906*k[6]+a0907*k[7]+a0908*k[8]),p,t+c8*dt))
-    copyat_or_push!(k,10,f(uprev+dt*(a1001*k[1]                                  +a1006*k[6]+a1007*k[7]+a1008*k[8]+a1009*k[9]),p,t+c9*dt))
-    copyat_or_push!(k,11,f(uprev+dt*(a1101*k[1]                                  +a1106*k[6]+a1107*k[7]+a1108*k[8]+a1109*k[9]+a1110*k[10]),p,t+c10*dt))
-    copyat_or_push!(k,12,f(uprev+dt*(a1201*k[1]                                  +a1206*k[6]+a1207*k[7]+a1208*k[8]+a1209*k[9]+a1210*k[10]+a1211*k[11]),p,t+c11*dt))
-    copyat_or_push!(k,13,f(uprev+dt*(a1301*k[1]                                  +a1306*k[6]+a1307*k[7]+a1308*k[8]+a1309*k[9]+a1310*k[10]+a1311*k[11]+a1312*k[12]),p,t+c12*dt))
-    copyat_or_push!(k,14,f(uprev+dt*(a1401*k[1]                                  +a1406*k[6]+a1407*k[7]+a1408*k[8]+a1409*k[9]+a1410*k[10]+a1411*k[11]+a1412*k[12]+a1413*k[13]),p,t+c13*dt))
-    copyat_or_push!(k,15,f(uprev+dt*(a1501*k[1]                                  +a1506*k[6]+a1507*k[7]+a1508*k[8]+a1509*k[9]+a1510*k[10]+a1511*k[11]+a1512*k[12]+a1513*k[13]+a1514*k[14]),p,t+dt))
-    copyat_or_push!(k,16,f(uprev+dt*(a1601*k[1]                                  +a1606*k[6]+a1607*k[7]+a1608*k[8]+a1609*k[9]+a1610*k[10]+a1611*k[11]+a1612*k[12]+a1613*k[13]),p,t+dt))
+    copyat_or_push!(k,2,f(uprev+dt*(a0801*k[1]                                  +a0806*k[6]+a0807*k[7]),p,t+c7*dt))
+    copyat_or_push!(k,3,f(uprev+dt*(a0901*k[1]                                  +a0906*k[6]+a0907*k[7]+a0908*k[2]),p,t+c8*dt))
+    copyat_or_push!(k,4,f(uprev+dt*(a1001*k[1]                                  +a1006*k[6]+a1007*k[7]+a1008*k[2]+a1009*k[3]),p,t+c9*dt))
+    copyat_or_push!(k,5,f(uprev+dt*(a1101*k[1]                                  +a1106*k[6]+a1107*k[7]+a1108*k[2]+a1109*k[3]+a1110*k[4]),p,t+c10*dt))
+    temp6 = recursivecopy(k[6])
+    temp7 = recursivecopy(k[7])
```

Comment:
Just out of curiosity: what's the advantage of copying `k[6]` and `k[7]` compared to just adding two additional elements to `k` (such that it's `copyat_or_push!(k,8,...)` up to `copyat_or_push!(k,12,..)` in the next lines)?

Reply:
Adding it to the `k` vector would increase the size of the saved vector.